package.json

```
@@ -115,7 +115,7 @@
     "prettier": "^2.7.1",
     "remark": "^14.0.2",
     "remark-cli": "^11.0.0",
-    "stylelint": "https://github.com/stylelint/stylelint/tarball/14.13.0",
+    "stylelint": "file:../stylelint",
```

Now it's possible to pass CI by using the HEAD version:

```suggestion
    "stylelint": "https://github.com/stylelint/stylelint/tarball/HEAD",
```